### PR TITLE
favicons

### PR DESCRIPTION
**Pull request by icy-td**

### CODE REVIEW

File: src/pages/index.jsx [hunk 1]
```diff
@@ -20,6 +20,9 @@ export default function Home() {
         />
         <script src="https://cdn.usefathom.com/script.js" data-site="GCGAUKHE" defer></script>
         <link rel="canonical" href="https://eloncodereview.com" />
+		<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+		<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+		<link rel="icon" type="image/png"  sizes="48x48" href="/favicon.png">
```
Comment (robzolkos) [inline src/pages/index.jsx:25]: I think these all need a closing link tag

**Review by robzolkos — CHANGES_REQUESTED** [review]

File: src/pages/index.jsx (hunk 1 above)
Comment (icy-td) [reply src/pages/index.jsx:25]: Nope! https://developers.google.com/search/docs/appearance/favicon-in-search

Comment (robzolkos) [reply src/pages/index.jsx:25]: With normal html you are correct.  With JSX however it needs to be closed.

**Comment by icy-td** [comment]
I'm an idiot, I forgot the '/'.

**Comment by robzolkos** [comment]
Thanks!

**Pull request merged**